bindings/http/http.go

```diff
@@ -318,6 +319,13 @@ func (h *HTTPSource) Invoke(parentCtx context.Context, req *bindings.InvokeReque
 
 		request.Header.Set(TracestateHeaderKey, ts)
 	}
+	if baggage, ok := req.Metadata[BaggageHeaderKey]; ok && baggage != "" {
+		if _, ok := request.Header[http.CanonicalHeaderKey(BaggageHeaderKey)]; ok {
+			h.logger.Warn("Tracing is enabled. A custom Baggage request header cannot be specified and is ignored.")
```

Comment:
Should this not break out of the outer if?

Reply:
The current implementation aligns with how `traceparent` and `tracestate` headers work (if you look above).